src/app/public/modules/input-box/fixtures/input-box.component.fixture.html

```diff
@@ -192,3 +192,31 @@
     </div>
   </sky-input-box>
 </div>
+
+<div
+  class="input-single-button-left"
+>
+  <sky-input-box>
+    <label
+      class="sky-control-label"
+      [for]="singleButtonLeftInput.id"
+    >
+      Input with single button on left
+    </label>
+    <div
+      class="sky-input-group-btn sky-input-box-btn-left"
+    >
+      <button
+        aria-label="Button 1"
+        class="sky-btn sky-btn-default sky-input-group-btn test-button-left"
```

Comment:
Does the `sky-input-group-btn` CSS class need to be on the `button` and the wrapping `div` element to work?

Reply:
Nope. Good catch.